src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -148,6 +148,28 @@
         }
       }
     },
+    "Sun": {
+      "source": "graphqldb.sun",
+      "permissions": [
+        {
+          "role": "anonymous",
+          "actions": [
+            "create",
+            {
+              "action": "read",
+              "fields": {
+                "include": [
+                  "*"
+                ]
+              }
+            },
```

Comment:
This section (which is valid) can also be represented as JSON string "read" which implies that all fields are included. That should behave the same way.